src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (let i = 0; i < people.length; i++) {
```

Comment:
```suggestion
people.forEach(({ name, sex, born, died, }) => {
```

Reply:
Better to use forEach and destructuring

Reply:
done, but there is a problem with eslint, i googled this problem and found, that it is a bug. 

If you are a user of TSLint-to-ESLint this was [a bug that has since been fixed](https://github.com/typescript-eslint/tslint-to-eslint-config/issues/856) so rerunning the script with a newer version would also fix the issue, or just disable the no-shadow and enable @typescript-eslint/no-shadow

So, some tests are not completed(